### PR TITLE
development and update for ITSQCtrackTask / Online Track Trending Task (solve conflicted)

### PR DESCRIPTION
**Pull request by delico18**
newly defined class
            ./QualityControl/Modules/ITS/include/ITS/TrendingTaskITSTracks.h
            ./QualityControl/Modules/ITS/src/TrendingTaskITSTracks.cxx   
            ./QualityControl/Modules/ITS/include/ITS/LinkDef.h       (modified)
            
modified class
            ./QualityControl/Modules/ITS/include/ITS/ITSTrackTask.h
            ./QualityControl/Modules/ITS/src/ITSTrackTask.cxx    
new plots are added.
    [TH1] distribution of the fraction of clusters in tracking event by event (ROF by ROF)
    [TH1] distribution of the number of tracks event by event (ROF by ROF)
    [TH2] number of clusters per track vs eta.

### CODE REVIEW

**Comment by iravasen** [comment]
@delico18 additional comments:

- please remove completely everything related to hClusterUsage and hOccupancyROF (remove also QC checks related to them). Now that we have distributions event by events, those histos are not needed anymore. Pay also attention to the field "MOs" in the json.

File: Modules/ITS/src/ITSTrackTask.cxx [hunk 1]
```diff
@@ -240,6 +245,25 @@ void ITSTrackTask::createAllHistos()
...
   formatAxes(hVertexContributors, "Number of contributors for vertex", "Counts", 1, 1.10);
   hVertexContributors->SetStats(0);
+  
+  hAssociatedClusterFraction = new TH1D("AssociatedClusterFraction", "AssociatedClusterFraction", 100, 0, 1);
+  hAssociatedClusterFraction->SetTitle("The fraction of clusters in tracking event by event");
+  addObject(hAssociatedClusterFraction);
+  formatAxes(hAssociatedClusterFraction, "", "Clusters in track / Clusters", 1, 1.10);
+  hAssociatedClusterFraction->SetStats(0);
+  
+  hNtracks = new TH1D("Ntracks", "Ntracks", mNtracksMAX, 0, mNtracksMAX);
```
Comment (iravasen) [inline Modules/ITS/src/ITSTrackTask.cxx:255]: Cast mNtracksMAX to "int" when using it for defining the number of bins. Not sure if it works like that.

**Comment by Barthelemy** [comment]
formatting issues to be fixed.

**Comment by iravasen** [comment]
@Barthelemy @knopers8 please close this PR. Opened a new one (https://github.com/AliceO2Group/QualityControl/pull/1008) which include all the commits here except the last one. Then it includes also few new commits from my side.

**Pull request closed without merging**